community/dbms/src/main/java/org/neo4j/server/configuration/ConfigLoader.java

```diff
@@ -61,6 +61,12 @@ public Config loadConfig( Optional<File> homeDir, Optional<File> configFile,
                 settingsClassesSupplier );
     }
 
+    public Config loadOfflineConfig( Optional<File> homeDir, Optional<File> configFile )
+    {
+        return loadOfflineConfig( homeDir, configFile,
+                Pair.of( GraphDatabaseSettings.auth_enabled.name(), Settings.FALSE ) );
+    }
+
```

Comment:
So why don't callers of this get the same behaviour: https://github.com/spacecowboy/neo4j/blob/e4e651d401a759136f1e9bfb1e32870c59674566/community/dbms/src/main/java/org/neo4j/server/configuration/ConfigLoader.java#L70?